notify/pagerduty/pagerduty.go

```diff
@@ -209,6 +209,11 @@ func (n *Notifier) notifyV2(
 		level.Debug(n.logger).Log("msg", "Truncated summary", "summary", summary, "key", key)
 	}
 
+	payloadSource := n.conf.Client
+	if n.conf.Source != "" {
+		payloadSource = n.conf.Source
+	}
```

Comment:
The default value could be handled in config/notifiers.go when unmarshaling the YAML data. This would be more explicit when users look at the /api/v2/status endpoint.

Reply:
Updated 👍